crypto/Cargo.toml

```diff
@@ -115,7 +115,7 @@ version = "0.5"
 features = ["async_std", "html_reports"]
 
 [dev-dependencies.core-crypto-attributes]
-version = "^1.0.0-rc.60"
+version = "^1.0.0"
```

Comment:
can you move this to the workspace dependency also?

Reply:
Sure, good catch!

Reply:
Done.